code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+	name = "Botanical Assistant"
+	accessParent = "Botanist"
+	canAuthorize = list("Botanist","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-hyd"
+	dispensedKit = /obj/item/storage/box/staffkit/botany
+
+
+//equipment kits
+
+/obj/item/storage/box/staffkit/med
+	name = "medical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/reagent_containers/emergency_injector/epinephrine,
+		/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/bandage)
+
+/obj/item/storage/box/staffkit/robo
+	name = "robotics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
+
+/obj/item/storage/box/staffkit/eng
+	name = "engineering assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/engineer,
+		/obj/item/clothing/head/helmet/hardhat,
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/mech
+	name = "mechanics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/mechanic,
+		/obj/item/electronics/soldering,
+		/obj/item/device/t_scanner,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/miner
+	name = "mining assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/clothing/head/emerg,
+		/obj/item/clothing/suit/space/emerg,
+		/obj/item/crowbar,
+		/obj/item/mining_tool/power_pick,
+		/obj/item/satchel/mining)
+
+/obj/item/storage/box/staffkit/res
+	name = "research assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffres,
+		/obj/item/device/radio/headset/research,
+		/obj/item/clothing/mask/gas,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/reagent_containers/glass/beaker,
+		/obj/item/device/reagentscanner)
+
+/obj/item/storage/box/staffkit/chef
+	name = "culinary assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffciv,
+		/obj/item/device/radio/headset/civilian,
+		/obj/item/clothing/head/souschefhat,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/satchel/hydro,
```

Comment:
Don't really like the free gloves out the gate.
Produce satchel is a neat thematic idea but it sorta feels like it is stealing from Botany. I think I'd rather see this more distributed on to more maps instead.

Reply:
Chefs wear gloves for sanitary reasons, but this is a thematic element that will not be played into by most people outside of RP. Including them by default gets them in the appropriate uniform.

Reply:
Latex gloves are a standard sous-chef item for food sanitation purposes and should probably remain. As to the other stuff, potential changes:
- Cleaning bottle swapped for sponge.
- Satchel swapped for one of the following: rolling pin, cookbook, apron. Leaning towards one of the latter two.

Reply:
Free gloves seems to just make forensics that much harder.  Most departments have gloves readily available to them already.  Thematically more of a "here are the items issued to you" vs "here is the equipment that is available".  You likely will "receive" a uniform, with extras in some locker, but the gloves are on the counter for people to put on.

Reply:
Forensics angle is a more serious consideration, and I think enough to warrant removal. Might swap satchel and gloves for cookbook and apron?

Reply:
No this isn't the place to change that balance, this is standard equipment and this is how the game is played. Worry about forensics in a forensics PR

Reply:
There's not really a big practical angle to latex gloves for culinary staff, and it buys room for an apron (which are I think actually less obtainable than latex gloves), so it's fine.